amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseCoordinatorIntegrationTest.java

```diff
@@ -48,21 +45,19 @@
 import software.amazon.kinesis.retrieval.kpl.ExtendedSequenceNumber;
 
 @RunWith(MockitoJUnitRunner.class)
-public class DynamoDBLeaseCoordinatorIntegrationTest {
-    private static final int ATTEMPTS = 20;
+public class DynamoDBLeaseCoordinatorIntegrationTest extends LeaseIntegrationTest {
```

Comment:
Now extends from `LeaseIntegrationTest`, so I could remove a whole bunch of test setup.